src/main/java/uk/gov/companieshouse/payments/admin/web/api/ApiClientService.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.payments.admin.web.api;
+
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.InternalApiClient;
+
+/**
+ * The {@code ApiClientService} interface provides an abstraction that can be
+ * used when testing {@code ApiClientManager} static methods, without imposing
+ * the use of a test framework that supports mocking of static methods.
+ */
+public interface ApiClientService {
+
+    ApiClient getApiClient();
```

Comment:
This can be removed as we're only going to use the internalApiClient for this admin service